testing/citest/tests/kube_smoke_test.py

```diff
@@ -527,6 +690,22 @@ def test_c_create_server_group(self):
   def test_d_create_find_image_pipeline(self):
     self.run_test_case(self.scenario.create_find_image_pipeline())
 
+  def test_cd_save__statefulsets_pipeline(self):
```

Comment:
As noted above, this will run after c but before d (not because of the d in the name, but because "c" < "cd" < "d". 

Also, do you intend the double _ here and below with the daemonset?